package.json

```diff
@@ -1,46 +1,48 @@
 {
-    "name": "next-dapp-starter-kit",
-    "version": "0.1.0",
-    "private": true,
-    "scripts": {
-        "dev": "next dev",
-        "build": "next build",
-        "start": "next start",
-        "lint": "next lint",
-        "graphql": "graphql-codegen --config codegen.ts"
-    },
-    "dependencies": {
-        "@fvictorio/newton-raphson-method": "^1.0.5",
-        "@radix-ui/react-scroll-area": "^1.0.5",
-        "@tanstack/react-query": "^5.36.2",
-        "big.js": "^6.2.1",
-        "class-variance-authority": "^0.7.0",
-        "connectkit": "^1.7.2",
-        "date-fns": "^3.6.0",
-        "graphql": "^16.8.1",
-        "graphql-request": "^6.1.0",
-        "next": "14.1.3",
-        "next-themes": "^0.3.0",
-        "react": "^18",
-        "react-dom": "^18",
-        "swapr-ui": "0.0.59",
-        "viem": "2.x",
-        "wagmi": "^2.5.7"
-    },
-    "devDependencies": {
-        "@graphql-codegen/cli": "5.0.2",
-        "@graphql-codegen/typescript": "4.0.6",
-        "@graphql-codegen/typescript-graphql-request": "^6.2.0",
-        "@types/big.js": "^6.2.2",
-        "@types/node": "^20",
-        "@types/react": "^18",
-        "@types/react-dom": "^18",
-        "autoprefixer": "^10.0.1",
-        "eslint": "^8",
-        "eslint-config-next": "14.1.3",
-        "postcss": "^8",
-        "prettier": "3.3.2",
-        "tailwindcss": "^3.3.0",
-        "typescript": "^5"
-    }
+  "name": "next-dapp-starter-kit",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "code:check": "npx prettier . --check",
+    "code:format": "npx prettier . --write",
```

Comment:
Let's not have this for now. We can add them if we need. This aren't currently being used right?

Reply:
I used them to format the codebase safely, but right now there is no pipeline or hook (like pre-commit or pre-push hooks) using them.

I'll remove them :D